### ISSUE TITLE
Add expandable inline alert

### ISSUE THREAD

**Issue by mcarrano** [patternfly/patternfly-react#5820]
Add expandable inline alert variant to Alert component. The user should be able to show/hide the contents of the alert by clicking the expansion caret.

https://github.com/patternfly/patternfly/issues/4056

**Comment by mcarrano** [comment]
@jessiehuff can you look at from an a11y perspective?

**Comment by jessiehuff** [comment]
I think that we'll likely just have to test this to make sure that focus shifts properly to the toggle and allows the user to expand the alert. I also think that it would probably require some documentation to explain the ways that this should and shouldn't be used. It also makes me wonder if this would be a good candidate to create a demo with. I think at one point we talked about creating a demo of an alert in a form, maybe this would fit into that?

**Comment by tlabaj** [comment]
closed by #6236
